src/foam/nanos/auth/PasswordHashingDAO.java

```diff
@@ -22,11 +26,53 @@ public PasswordHashingDAO(X x, DAO delegate) {
   }
 
   @Override
+  public Sink select_(X x, Sink sink, long skip, long limit, Comparator order, Predicate predicate) {
+    if ( sink == null ) sink = new ListSink();
```

Comment:
Weren't you going to handle this in AbstractDAO.

Reply:
It isn't working because the DAOSkeleton calls select_ instead of select